src/engraving/dom/segment.cpp

```diff
@@ -1438,6 +1438,8 @@ EngravingItem* Segment::elementAt(track_idx_t track) const
 
 void Segment::scanElements(void* data, void (* func)(void*, EngravingItem*), bool all)
 {
+    all = all || (isType(SegmentType::TimeSig | SegmentType::TimeSigAnnounce)
+                  && style().styleV(Sid::timeSigPlacement).value<TimeSigPlacement>() != TimeSigPlacement::NORMAL);
```

Comment:
I'm not sure if modifying `all` is a good idea because it gets passed to the recursive `scanElements` calls. Or is that exactly the intention?